pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.black]
```

Comment:
(This is just a suggestion to get rid of extra configuration files. Feel free to ignore.)

One more thing, the purpose of using `pflake8` instead of `flake8` is to be able to put the configurations in the `pyproject.toml` file. since we're using [`pflake8`](https://github.com/csachs/pyproject-flake8) we can just move the settings from `.flake8` to `pyproject.toml` (under `[tool.pflake8]` section, I suspect) and get rid of the `.flake8` file.